airbyte-webapp/src/hooks/services/useConnectionHook.tsx

```diff
@@ -137,13 +137,16 @@ const useCreateConnection = () => {
         ...values,
         status: "active",
         sourceCatalogId,
+
+        // TODO(harshith): Remove this once backend starts accepting only scheduleData params
+        schedule: values.scheduleData?.basicSchedule,
```

Comment:
We never check these TODOs. Can you create an issue in github describing what needs to be done?

Reply:
I moved completely to `scheduleData` and `scheduleType` and eliminated schedule out of frontend code